plugins/metrics/OTEL_OPENINFERENCE_SUPPORT.md

```
@@ -0,0 +1,637 @@
+
+Build an enterprise observability layer around Claude Code CLI without modifying the Claude Code codebase. The goal is to measure:
+
+* How effectively Claude Code uses company agentic documentation.
+* How documentation influences agent behavior, decisions, and outcomes.
+* Agent reasoning and execution efficiency.
+* Model inference performance.
+* Tool usage patterns.
+* Failure modes, loops, and wasted execution.
+* Cost, latency, token utilization, and engineering productivity impact.
+
+The implementation should use Claude Code’s existing telemetry capabilities, OpenTelemetry transport, OpenInference semantic conventions, and external observability infrastructure. Claude Code currently supports exporting OpenTelemetry metrics, logs/events, and traces through OTLP, including model requests, tool execution, tokens, cost, sessions, and agent execution spans. ([Claude Code][1]) OpenInference should be used as an AI observability semantic layer on top of OpenTelemetry to standardize LLM, retrieval, and tool-related tracing concepts. ([Arize AI][2])
```

markdownlint-cli2 (0.22.1)

[warning] Reference links and images should use a label that is defined
Missing link or image reference definition: "1"

(MD052, reference-links-images)